### DISCUSSION TITLE
parameters - ' --models-dir MODELS_DIR', How to type in exactly? all my models in E:\StableDiffusionModels. Please help the old man(65y.o.).

### DISCUSSION THREAD

**Discussion by bccafe66** [vladmandic/sdnext#2666] [Q&A]
parameters - ' --models-dir MODELS_DIR', How to type in exactly? all my models in E:\StableDiffusionModels. Please help the old man(65y.o.).
They show me in A1111, set COMMANDLINE_ARGS=--theme dark --xformers --no-half-vae --api --medvram-sdxl --enable-insecure-extension-access --embeddings-dir "K:\StableDiffusionWebui\stable-diffusion-webui\embeddings" --ckpt-dir "E:\StableDiffusionModels\Stable-diffusion" --lora-dir "E:\StableDiffusionModels\Lora" 
But no example in here, so somebody help me please.

**Comment by vladmandic** [comment]
just use ui -> settings -> system paths

**Comment by brknsoul** [comment]
`webui.bat --models-dir E:\StableDiffusionModels`

This assumes you have folders such as `lora`, `embeddings`, `Stable-diffusion`, etc, etc in `E:\StableDiffusionModels`

**Reply by brunogcar** [reply]
not this case but, if you have space in folder name, in windows you need to add `" "` like `"E:\Stable Diffusion Models"`
so  in  general i would recommend to add `--models-dir "E:\StableDiffusionModels"`
